braindecode/datasets/base.py

```diff
@@ -385,10 +384,8 @@ def save(self, path, overwrite=False):
         description_file_name = os.path.join(path, 'description.json')
         target_file_name = os.path.join(path, 'target_name.json')
         if not overwrite:
-            if (os.path.exists(description_file_name) or
-                    os.path.exists(target_file_name)):
-                raise FileExistsError(
-                    f'{description_file_name} or {target_file_name} exist in {path}.')
+            from braindecode.datautil.serialization import check_save_dir_empty
```

Comment:
you need this to avoid a circular import?

Reply:
Yes, that's right.